src/custom/pages/About/index.tsx

```diff
@@ -1,7 +1,82 @@
 import React from 'react'
-import content from 'pages/About/About.md'
-import Markdown from 'components/Markdown'
+import Page, { Title, Content } from 'components/Page'
+
+// Assets
+import diagramIMG from 'assets/cow-swap/cowswap-diagram.png'
+import gaslessIMG from 'assets/cow-swap/gasless.png'
+import mevIMG from 'assets/cow-swap/mev.png'
 
 export default function About() {
-  return <Markdown content={content} />
+  return (
+    <Page>
+      <Title>About</Title>
+
+      <Content>
+        <small>(updated: April 2021)</small>
```

Comment:
Are we sure we want this? Fine with me, just double checking since I don't usually see it in the FAQs. 

It made sense for the policies